src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -2794,35 +2678,6 @@
                 "type": "query",
                 "useTags": false
             },
-            {
```

Comment:
same here

Reply:
We're not using this filter anymore as it's a duplicate of `juju_application`.